wren/compute/manager.go

```diff
@@ -76,12 +76,14 @@ func (m *Manager) CompileQuery(ctx context.Context, owner *ent.Owner, query stri
 	subLogger := log.Ctx(ctx).With().Str("method", "manager.CompileQuery").Logger()
 	formulas, err := m.getFormulas(ctx, owner, requestViews)
 	if err != nil {
-		return nil, errors.WithMessage(err, "getting views")
+		subLogger.Error().Err(err).Msg("issue getting formulas")
+		return nil, err
```

Comment:
Is removing the error wrapper intentional?

Reply:
yes. the error wrapper package has been deprecated for quite some time and it recommended for use anymore.